src/components/inputs/DateInput/DateInput.scss

```diff
@@ -0,0 +1,14 @@
+$C: #{$WEBEX_COMPONENTS_CLASS_PREFIX}-date-input;
+
+.#{$C} {
+  input[type="date"] {
+    position: relative;
+    padding-left: 2.5rem;
+
+    &::-webkit-calendar-picker-indicator {
+      position: absolute;
+      left: -1rem;
+      filter: var(--wxc-input-calendar-icon--color);
```

Comment:
the name should have the --filter suffix not to confuse anybody to think this can have a color value

and let's keep the convention of --wxc-<component>--<prop>

that is `--wxc-date-input--calendar-picker-indicator--filter`